### PR TITLE
Covered Francisco's blog post on migrating his app to Ember

### PR DESCRIPTION
**Pull request by ijlee2**

### CODE REVIEW

File: source/2020-04-24-the-ember-times-issue-145.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-author: Chris Ng, Matthew Roderick, Jared Galanis the crowd
+author: Chris Ng, Matthew Roderick, Jared Galanis, Isaac Lee the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
 alias : "blog/2020/04/24-the-ember-times-issue-145.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 Simple drag and drop in Ember Octane 🐺,
+Meet 2 developers who became an Ember developer 🎉,
```
Comment (ijlee2) [inline source/2020-04-24-the-ember-times-issue-145.md:11]: I will later write about Jenny's blog post on her learning experience.

File: source/2020-04-24-the-ember-times-issue-145.md [hunk 2]
```diff
@@ -44,14 +45,13 @@ To see any of the individual changes, please look at the pre-release versions in
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Migré de Svelte a EmberJS y Fue Para Bien 🎉](https://otroespacioblog.wordpress.com/2020/04/17/migre-de-svelte-a-emberjs-y-fue-para-bien/)
```
Comment (ijlee2) [inline source/2020-04-24-the-ember-times-issue-145.md]:
```suggestion
## [Migré de Svelte a EmberJS y fue para bien 🎉](https://otroespacioblog.wordpress.com/2020/04/17/migre-de-svelte-a-emberjs-y-fue-para-bien/)
```

File: source/2020-04-24-the-ember-times-issue-145.md [hunk 3]
```diff
@@ -44,14 +45,13 @@ To see any of the individual changes, please look at the pre-release versions in
...
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Say hello to [Francisco Quintero (@cesc1989)](https://github.com/cesc1989). Francisco is a software engineer who recently [migrated his app Intrati from Svelte to Ember Octane](https://otroespacioblog.wordpress.com/2020/04/17/migre-de-svelte-a-emberjs-y-fue-para-bien/) in **16 days**! In both cases, a friend recommended the framework and he learned by reading documentation, writing tests, and coding by trial-and-error.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In the blog post, Francisco shares what he liked and what he found to be difficult to do in Svelte and Ember. He also shares [the notes that he compiled while learning Ember](https://paper.dropbox.com/doc/Aprendizaje-Migrando-a-EmberJS--AyOEl5onXX29iRjnY_mtdqygAQ-Gj5CGvslVsQcrmGMKvWtR). We encourage you to read them to understand more what using Ember can be like for a new developer.
```
Comment (ijlee2) [inline source/2020-04-24-the-ember-times-issue-145.md]:
```suggestion
In the [blog post](https://otroespacioblog.wordpress.com/2020/04/17/migre-de-svelte-a-emberjs-y-fue-para-bien/), Francisco shares what he liked and what he found to be difficult to do in Svelte and Ember. He also shares [the notes that he compiled while learning Ember](https://paper.dropbox.com/doc/Aprendizaje-Migrando-a-EmberJS--AyOEl5onXX29iRjnY_mtdqygAQ-Gj5CGvslVsQcrmGMKvWtR). We encourage you to read them to understand more what using Ember can be like for a new developer.
```

File: source/2020-04-24-the-ember-times-issue-145.md [hunk 4]
```diff
@@ -44,14 +45,13 @@ To see any of the individual changes, please look at the pre-release versions in
...
+Say hello to [Francisco Quintero (@cesc1989)](https://github.com/cesc1989). Francisco is a software engineer who recently [migrated his app Intrati from Svelte to Ember Octane](https://otroespacioblog.wordpress.com/2020/04/17/migre-de-svelte-a-emberjs-y-fue-para-bien/) in **16 days**! In both cases, a friend recommended the framework and he learned by reading documentation, writing tests, and coding by trial-and-error.
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+In the [blog post](https://otroespacioblog.wordpress.com/2020/04/17/migre-de-svelte-a-emberjs-y-fue-para-bien/), Francisco shares what he liked and what he found to be difficult to do in Svelte and Ember. He also shares [the notes that he compiled while learning Ember](https://paper.dropbox.com/doc/Aprendizaje-Migrando-a-EmberJS--AyOEl5onXX29iRjnY_mtdqygAQ-Gj5CGvslVsQcrmGMKvWtR). We encourage you to read them to understand more what using Ember can be like for a new developer.
+
+(Both Francisco's blog post and notes are written in Spanish. If you're not familiar with the language, you can use Google Translate or another translation software.)
```
Comment (jayjayjpg) [inline source/2020-04-24-the-ember-times-issue-145.md:54]: :sparkles:

**Review by jayjayjpg — APPROVED** [review]
This reads great! Thank you for another :sparkles: writeup for the Ember Times this week!

**Pull request merged**